modules/statics/src/coins.ts

```diff
@@ -1579,14 +1579,6 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.SRM,
     AccountCoin.DEFAULT_FEATURES
   ),
-  solToken(
```

Comment:
These are breaking changes and need to be added to the commit footer.

Reply:
Updated the message commit, so how does breaking change gonna be handled later?

Since in my case I need to change the available sol token asset in wallet-platform after updating the static lib and do the change in indexer as well (which I already have PR open for that)

Reply:
It will cause a `major` version bump in the next release. You'll need to fix BREAKING CHANGE to be in the commit footer.

Reply:
You'll still keep the BG-number on the next line after the description in the footer.

Reply:
```
BREAKING CHANGE: This breaks the provided token asset used for wp in bitgo-ms,
will require to change available solana token asset in wp once this merged.
BG-52918
```